src/pretalx/orga/forms/review.py

```diff
@@ -41,20 +56,32 @@ def clean(self):
 
 class ReviewForm(ReadOnlyFlag, forms.ModelForm):
 
-    def __init__(self, event, user, *args, **kwargs):
+    def __init__(self, event, user, *args, instance=None, **kwargs):
         self.event = event
-        super().__init__(*args, **kwargs)
-        min_value = int(event.settings.review_min_score)
-        max_value = int(event.settings.review_max_score)
+        self.may_override = event.settings.allow_override_votes and user.remaining_override_votes(event)
+        self.may_override = self.may_override or (instance and instance.override_vote is not None)
+        self.min_value = int(event.settings.review_min_score)
+        self.max_value = int(event.settings.review_max_score)
+        if instance:
+            if instance.override_vote is True:
+                instance.score = self.max_value + 1
```

Comment:
is there a way to make this a calculated property and give it a proper name like `event.settings.review_override_score`? There are 7 places across 2 files, where this is calculated.

Reply:
I'm a bit hesitant to add more and more properties to the event class, but I guess you're right. I'll add it later. Any other things I should look for?